test/sequential/test-net-connect-emfile.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+
+console.log('Test to check the error message format during EMFILE error');
+
+if (process.platform === 'win32') {
+  console.log('Skipping test, no RLIMIT_NOFILE on Windows.');
+  return;
+}
+
+var totalErrorCount = 0;
+
+const reactToError = function() {
+  totalErrorCount += 1;
+  if (totalErrorCount === 2) {
+    console.log('Test OK');
+    process.exit(0);
```

Comment:
Awesome, this is a great suggestion :-) Thanks :-) I am using `common.mustCall` now